src/test/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidatorUTest.java

```diff
@@ -52,4 +53,84 @@ public void validate_rejectMinutesPerSessionGreaterThanTotalMinutes() {
         assertEquals("ExceedTotal.courseConfigurationForm.defaultMinutesPerSession", capturedErrorCode.getValue());
     }
 
+    @Test
+    public void validate_rejectWrongPoints() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)